backend/substrapp/tasks/tasks.py

```diff
@@ -704,9 +708,8 @@ def _do_task(client, subtuple_directory, tuple_type, subtuple, compute_plan_id,
                 except kubernetes.client.rest.ApiException as e:
                     logger.error(f'failed to remove secrets from namespace {secret_namespace}')
                     raise e
-            logger.info(f'{len(secrets)} secrets have been removed')
-
-        volumes[chainkeys_directory] = {'bind': '/sandbox/chainkeys', 'mode': 'rw'}
+            else:
+                logger.info(f'{len(secrets)} secrets have been removed')
```

Comment:
the idea here was to replace the secrets before making them available to the task in a volume, instructions order is important here. Maybe we should add a comment.

Reply:
I don't understand you comment. Secrets are retrieve from k8s secret and then dump into file in the volume which will be binded.
The instruction here, only erase secret value. I add the log in the else (for .. else loop) to make it explicit that logger is trigger if and only if we manage to do this because we raise if we don't. It does the same stuff than before, it just more explicit.

Reply:
I don't understand you comment. Secrets are retrieve from k8s secret and then dump into file in the volume which will be binded.
The instruction here, only erase secret value. I add the log in the else (for .. else loop) to make it explicit that logger is trigger if and only if we manage to do this because we raise if we don't. It does the same stuff than before, it just more explicit.